### ISSUE TITLE
Ability to use your own classes

### ISSUE THREAD

**Issue by SiamKreative** [gambitph/Page-Builder-Sandwich#8]
Hey Benjamin,

I wanted to create something like this for a while. As mentioned, I'm currently using [Visual Columns for WordPress](http://codecanyon.net/item/visual-columns-for-wordpress/6505510) but I'm happy to make the switch any time soon.

What I would like to be able to do is to be able to define my own classes representing columns. For instance, if my site is built with Boostrap 3, I'd like to disable the plugin's stylesheet (at this time there's no stylesheet as it's [inline styling](https://github.com/gambitph/Shortcodeless-Columns/issues/7)) I'd like to have the ability to define my columns, like so:

* 1/4 = col-lg-3
* 1/3 = col-lg-4
* 2/3 = col-lg-8
* 1/2 = col-lg-6
* etc etc

This doesn't have to be on a settings page. I'm happy with PHP or else.

What do you think?

**Comment by bfintal** [comment]
I'd like to add this option inside a settings modal window for the columns in #5

**Comment by SiamKreative** [comment]
I see that you decided to use namespaced Boostrap classes for the columns (and a few other things). This is great as I'm using Boostrap 3 in most of my themes. But what's the proper way to remove the plugin's stylesheet?

Also, in the past I used to namespace Bootstrap as well. But lately, I have used prefixes instead as I found out there could be possible conflicts:

* If the namespaced stylesheet uses version 3, it can still conflict if the theme is based on BS2.3.2
* If the namespaced stylesheet uses version 2, it can still conflict if the theme is based on BS3

So it would be great clearly identify the pros/cons of namespacing in the context of using it inside a WordPress plugin.
